### ISSUE TITLE
Question: busy day and array of not busy time ranges

### ISSUE THREAD

**Issue by sazanof** [fullcalendar/fullcalendar#3643]
thank you for the great addition!

tell me, is it possible to deduce some indication that the day is filled with events?

is it possible to get an array of free hours and hours are already employed for a specific day?

**Comment by arshaw** [comment]
No, not yet conveniently possible. This request is related to #3217, but it is a distinct request because you'd want to get an array of *available* times.

I understand the need for this, thanks.

**Comment by fjparedesb** [comment]
Hi, zasanof did you manage to find a workaround this?

**Comment by aamirco** [comment]
Hi guys, was there any movement on this feature? Ive been scratching my head over it all day.. 
Any update would be much appreciated !
